dask_cuda/benchmarks/local_cudf_shuffle.py

```diff
@@ -129,7 +129,34 @@ def bench_once(client, args, write_profile=None):
         ctx = performance_report(filename=write_profile)
 
     with ctx:
-        if args.backend in {"dask", "dask-noop"}:
+        bootstrapped = False
+        if args.backend == "rapidsmpf":
+            from rapidsmpf.config import Options
+            from rapidsmpf.examples.dask import dask_cudf_shuffle
+            from rapidsmpf.integrations.dask import bootstrap_dask_cluster
+
+            options = Options({"dask_spill_device": "0.5", "dask_statistics": "true"})
+            if not bootstrapped:
+                shuffle_plugin = client.run(
+                    lambda dask_worker: "shuffle" in dask_worker.plugins
+                ).values()
+                if not all(list(shuffle_plugin)):
+                    bootstrap_dask_cluster(client, options=options)
+                    bootstrapped = True
+            t1 = perf_counter()
+            partition_count = args.in_parts
+
+            shuffled = dask_cudf_shuffle(
+                df,
+                ["data"],
+                sort=False,
+                config_options=options,
+                partition_count=partition_count,
+            )
+            wait(shuffled.persist())
+            duration = perf_counter() - t1
+            assert shuffled.npartitions == partition_count
```

Comment:
Can we make this consistent with other implementations and move this into a `shuffle_rapidsmpf`?